init-dev-DB.sh

```diff
@@ -11,6 +11,65 @@ else
   echo Looks like PostgreSQL is already installed.
 fi
 
+# ------------------------------------------------------------------------------
+# The next section repairs possible missing folders whose absence prevent
+# Postgres from starting correctly, This problem occurs with the latest
+# versions of macOS (e.g. Yosemite, El Capitan and Sierra).
+
+if [ ! -d "/usr/local/var/postgres/pg_tblspc" ]; then
```

Comment:
Maybe this can be done a little less verbosely by iterating over a list (of files) and using that to check if it exists and if not creating it?